empress/topoSort/test_ReconBuilder.py

```diff
@@ -0,0 +1,150 @@
+import unittest
+import os
+import itertools
+import shutil
+from empress import newickFormatReader
+from empress.topoSort import ReconBuilder
+from empress.clumpr import DTLReconGraph
+from empress.clumpr import HistogramAlgTools
+from empress.clumpr.script02_gen_newick_trees import generateNewickTestsMultipleSizes
+
+class TestReconBuilder(unittest.TestCase):
+    """
```

Comment:
Write a comment that says that most trees in the bottom test is temporally consistent, while the inconsistent trees are tested separately.